lib/interface_quda.cpp

```diff
@@ -361,12 +362,52 @@ static int qmp_rank_from_coords(const int *coords, void *fdata)
 #endif
 
 
+// Provision for user control over MPI comm handle
+// Assumes an MPI implementation of QMP
+
+MPI_Comm MPI_COMM_HANDLE;
```

Comment:
This will require some `#ifdef` to not break compilation if we compile without MPI.